app/models/sample.rb

```diff
@@ -44,7 +44,7 @@ def initiate_s3_cp
     fastq2 = input_files[1].source
     command = "aws s3 cp #{fastq1} #{sample_input_s3_path}/;"
     command += "aws s3 cp #{fastq2} #{sample_input_s3_path}/;"
-    if s3_preload_result_path.present?
+    if s3_preload_result_path.size > 5 && s3_preload_result_path[0..4] == 's3://'
```

Comment:
you can skip the first part of this condition. ruby will not error when you slice longer than the string

Reply:
ok cool~